lib/src/rules/always_specify_types.dart

```diff
@@ -102,6 +104,18 @@ class AlwaysSpecifyTypes extends LintRule implements NodeLintRule {
 }
 
 class _Visitor extends SimpleAstVisitor<void> {
+  static const LintCode keywordCouldBeTypeCode = LintCode(
+      "always_specify_types", "'{0}' could be '{1}'.",
+      correction: "Try changing the type.");
+
+  static const LintCode keywordCouldBeSplitToTypesCode = LintCode(
+      "always_specify_types", "'{0}' could be split into types.",
+      correction: "Try splitting '{0}' to different types.");
```

Comment:
In addition to the comments above, this one is a bit confusing because I don't know what it means to split a keyword into types. I think what you're trying to say is that the declaration could be split into multiple declarations in order to be able to provide more specific type information. That's a valid way to correct the problem and definitely one that we should suggest, but let's talk about splitting the declaration rather than splitting a keyword. (And, of course, the problem isn't that the declarations could be split up.)

Reply:
Forgive my inaccuracy. Hopefully fixed.